poms/alfresco-sdk-parent/pom.xml

```diff
@@ -231,8 +233,8 @@
                     <artifactId>maven-compiler-plugin</artifactId>
                     <version>${maven.compiler.version}</version>
                     <configuration>
-                        <source>1.7</source>
-                        <target>1.7</target>
+                        <source>${maven.compiler.source}</source>
```

Comment:
These 2 lines are unneeded since maven.compiler.source and maven.compiler.target are the user properties so can be configured in <properties> and <properties> only.